Automattic/wp-calypso
# E2E: Check if Help Center opens correctly in Calypso and the Editor

Related to p7DVsv-kMu-p2

## Proposed Changes

* E2E tests to ensure the Help Center opens correctly within Calypso and the editor.

## Why are these changes being made?


* There was an issue with the Help Center not opening in the editor due to a removed scope (`PinnedItems/core/edit-post`) in the WP 6.5 upgrade. E2E tests are necessary to catch any potential issues related to the Help Center opening before they reach production.

## Testing Instructions



* Checkout this branch
* Follow the E2E setup instructions if you haven't already (https://github.com/Automattic/wp-calypso/tree/trunk/test/e2e)
* Verify all the Help Center related E2E tests pass by running: `yarn workspace @automattic/calypso-e2e build && yarn jest specs/help-center/*`

## Pre-merge Checklist
